src/components/Credits/index.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import { Box, Grid, Typography } from '@material-ui/core'
+
+import sponsorsData from '../../../data/credits/sponsors.json'
+import partnersData from '../../../data/credits/partners.json'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    padding: '96px 48px',
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: theme.palette.background.secondary,
+    flexDirection: 'column',
+    [theme.breakpoints.down('xs')]: {
+      padding: '48px',
+    },
+  },
+  title: {
+    marginBottom: 24,
+  },
+  sectionWrapper: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flexDirection: 'column',
+    marginTop: 20,
+    [theme.breakpoints.down('xs')]: {
+      marginTop: 32,
+    },
+  },
+  sectionHeader: {
+    fontWeight: 600,
+    [theme.breakpoints.down('xs')]: {
+      fontSize: 20,
```

Comment:
Can we increase the size of the headers in mobile? Think 32px will be good

Reply:
22b8520942bac69d7c64469c4f12f8620d9d7ae6